ironfish/src/network/peerNetwork.test.ts

```diff
@@ -767,7 +775,14 @@ describe('PeerNetwork', () => {
         const { wallet, memPool, chain } = node
         const accountA = await useAccountFixture(wallet, 'accountA')
         const accountB = await useAccountFixture(wallet, 'accountB')
-        const { transaction } = await useBlockWithTx(node, accountA, accountB)
+        const { transaction } = await useBlockWithTx(
```

Comment:
Updating fixtures here. This test should not sync to mempool when creating this transaction